packages/design-system/src/stitches.config.ts

```diff
@@ -271,6 +271,9 @@ export const {
       4: "400",
       max: "999",
     },
+    columns: {
+      1: "40% 60%",
```

Comment:
I am not quite sold about the reusability of this token, it sounds like a specific case, am I wrong? If yes please tell me how it can be reused everywhere